static/js/components/ExpandedLearningResourceDisplay.js

```diff
@@ -360,6 +362,16 @@ export default function ExpandedLearningResourceDisplay(props: Props) {
           <PaginatedUserListItems userList={object} pageSize={10} />
         </CollapsableSection>
       ) : null}
+      {object.object_type === LR_TYPE_PODCAST ? (
+        <CollapsableSection
```

Comment:
Not sure if this is an issue with your code in this PR or not, but when the episode list is expanded, the collapsible panel icon is still a plus sign.  Shouldn't it be a minus sign?

![Screen Shot 2020-04-29 at 2 02 00 PM](https://user-images.githubusercontent.com/12089658/80630416-2f947100-8a22-11ea-9ae2-44ea0a311cd6.png)

Reply:
this is an existing component I'm re-using, but I will fix the bug

Reply:
Feel free to do that in a separate PR if it's unrelated, I just wasn't sure.

Reply:
yes